tests/drivers/can/timing/src/main.c

```diff
@@ -40,18 +40,18 @@ static const struct can_timing_test can_timing_tests[] = {
 	/** Standard bitrates. */
 	{  125000, 875, false },
 	{  500000, 875, false },
-	{ 1000000, 875, false },
+	{ 1000000, 750, false },
```

Comment:
For diversification?

Reply:
No. As described in the commit log, these sample point locations match those used by the Linux kernel and seem to be much more likely to be met with 100% accuracy by a wide range of CAN controllers.